sdks/java/io/hadoop-format/build.gradle

```diff
@@ -20,7 +20,7 @@ import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
 applyJavaNature(
-  enableChecker:false,
+
```

Comment:
nit: some of these are left with unnecessary whitespace (not a big deal if its a pain to fix)